airbyte-webapp/src/components/JobItem/components/MainInfo.module.scss

```diff
@@ -8,12 +8,14 @@
   .titleCell {
     display: flex;
     color: colors.$dark-blue;
+    min-width: 0;
```

Comment:
In order to keep the content of this cell from allowing it to overflow its parent.  Same thing below in `.justification`